torch/ao/quantization/fx/prepare.py

```diff
@@ -159,13 +159,23 @@ def _is_input_arg_dtype_supported_by_backend(
             dtype_config, backend_config) for a in arg)
     if not isinstance(arg, Node):
         return True
+    assert "input_obs_or_fq_ctr_for_args" in node.meta["target_dtype_info"]
+    input_obs_or_fq_ctr_for_args = node.meta["target_dtype_info"]["input_obs_or_fq_ctr_for_args"]
+    arg_to_ctr = {}
+    _populate_arg_to_ctr_for_args(node.args, input_obs_or_fq_ctr_for_args, arg_to_ctr)
```

Comment:
I also presume this will be simplified because right now we seem to be doing this check `_is_input_arg_dtype_supported_by_backend` for every arg of a node. Anyways with new API, this should not be needed, right?

Reply:
yeah, some of this are kept for bc purposes, with the new API and improved BackendConfig, we don't need to have separate checkings for act/weight/bias